src/toolkit/metadata.cycpp.h

```diff
@@ -0,0 +1,33 @@
+
+// This includes the required header to add geographic coordinates to a
```

Comment:
Is this a cut & paste error - this comment is about metadata and not geo coordinates